README.md

```diff
@@ -220,6 +220,25 @@ A volume can also be used to map this directory to one external to the container
   docker run -d -p 8080:80 -p 8021:21 -e "GALAXY_LOGGING=full" -v `pwd`/gx_logs:/home/galaxy/logs bgruening/galaxy-stable
   ```
 
+Integrating tools non-Tool Shed tools into the container
+--------------------------------------------------------
+
+We recommend to use the [Main Galaxy Tool Shed](https://toolshed.g2.bx.psu.edu/) for all your tools and workflows that you would like to share.
+In rare situations where you cannot share your tools but still want to include them into your Galaxy Docker instance, please follow the next steps.
+
+ * Get your tools into the container.
+
+    You can copy your tools into a separate folder under `/home/user/galaxy_storage/` let's say `/home/user/galaxy_storage/local_tools`, or you can mount your tool directory into the container with a separate `-v /home/user/my_galaxy_tools/:/local_tools`.
+
+ * Create a `tool_conf.xml` file for your tools. 
+ 
+    This should look similar to the main [`tool_conf.xml`](https://github.com/galaxyproject/galaxy/blob/dev/config/tool_conf.xml.sample) file, but references your tools from the new directory. In other words a tool entry should look like this `<tool file="/local_tools/application_foo/foo.xml" />`.
+    This file can be located in `/local_tools` or `/export/local_tools` depending on your setup. The only important part is, it should be available from inside of the container. We assume you have it stored under `/local_tools/my_tools.xml`.
```

Comment:
As `export` is not mention before, it can be difficult to determine which is the current setup and how to manage the `.xml` files